recipes/phcue-ck/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -euo
```

Comment:
This looks weird. `-o` is normally followed by something to set, such as `-o pipefail`.

Reply:
Yes... Thank you! I had copy pasted from another rust recipe and did not notice it!